rb_ws/src/buggy/launch/sim_2d.launch

```diff
@@ -0,0 +1,22 @@
+<!-- roslaunch buggy main.launch simulation:=true -->
```

Comment:
I think this file is deprecated? Please remove if so. I think [sim_2d_2buggies.launch](https://github.com/CMU-Robotics-Club/RoboBuggy2/blob/main/rb_ws/src/buggy/launch/sim_2d_2buggies.launch) and [sim_2d_single.launch](https://github.com/CMU-Robotics-Club/RoboBuggy2/blob/main/rb_ws/src/buggy/launch/sim_2d_single.launch) are the new ones. @Jackack  please confirm

Reply:
This is indeed deprecated